packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_inputs.dart

```diff
@@ -208,22 +208,25 @@ mixin _PropertyInputMixin<T> {
   }
 
   Widget inputLabel(EditableProperty property, {required ThemeData theme}) {
+    final fixedFontStyle = theme.fixedFontStyle.copyWith(
+      fontSize: defaultFontSize + 1,
```

Comment:
can you add a comment explaining why we need to increase the font size?

Reply:
Done!